src/applications/pensions/config/chapters/02-military-history/servicePeriod.js

```diff
@@ -73,7 +77,7 @@ export default {
     type: 'object',
     required: ['serviceBranch', 'activeServiceDateRange'],
     properties: {
-      serviceBranch: checkboxGroupSchema(Object.keys(serviceBranchLabels)),
+      serviceBranch,
```

Comment:
Why did we revert the service branch schema to the vets-json-schema? This decouples the 'checkboxGroupUI' web component pattern from it's schema.